<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_job_title</class>
 <widget class="QMainWindow" name="new_job_title">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastrar Função</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(82, 152, 234);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nome da Função</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="3">
     <widget class="QTableWidget" name="jobTitle_tableWidget">
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Função</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Descrição</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="name_lineEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Descrição da Função</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="description_lineEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="addTitle_pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Adicionar Função</string>
      </property>
      <property name="icon">
       <iconset resource="icons.qrc">
        <normaloff>:/Icons/add.svg</normaloff>:/Icons/add.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>832</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuA_es">
    <property name="title">
     <string>Ações</string>
    </property>
    <addaction name="actionSalvar"/>
    <addaction name="actionSair"/>
   </widget>
   <addaction name="menuA_es"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(82, 152, 234);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="iconSize">
    <size>
     <width>42</width>
     <height>42</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSalvar"/>
  </widget>
  <action name="actionSalvar">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/Icons/save.svg</normaloff>:/Icons/save.svg</iconset>
   </property>
   <property name="text">
    <string>Salvar Cadastro</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
